lib/phoenix/endpoint.ex

```diff
@@ -425,6 +425,11 @@ defmodule Phoenix.Endpoint do
   """
   @callback static_path(path :: String.t) :: String.t
 
+  @doc """
+  Generates an integrity hash to a static file in `priv/static`.
+  """
+  @callback static_integrity(path :: String.t) :: String.t
```

Comment:
It can also return `nil`. We should also document the static_lookup callback.

Reply:
Done